File: Python/PB8.py
#From The given txt file find the number of lnes starting with "From" and extract the mail id's from that line
# Calculate the total mail id's 
f=open("email.txt","rt")
data=f.read()
people_who_sent_mail=[]
for line in f:
    if not line.startswith('From: '): continue
    pieces = line.split()
    email = pieces[1]
    people_who_sent_mail.append(email)
print(len(people_who_sent_mail))
f.close()

File: C++/PB8.cpp
//Find the total Number of lines In the text file
#include <iostream>
#include <regex>
#include <fstream>
#include <string>
#include <string_view>
#include <regex>
#include <sstream>
using namespace std;
int main() {
    string myText;
    int c=0;
    ifstream emailfile("email.txt");
    if (emailfile.is_open())
    {
        cout<<"Reading From File Operation Started!"<<endl;
        for( string line getline( emailfile, myText ); )
        {
            c=c+1;
        }
        cout<<"The Number Of Lines In the File are:"<<endl;
        cout<<c<<endl;
        emailfile.close();
    }
    else
    {
        std::cout << "File not opened." << std::endl;
    }

    return 0;
}
